project/PACKAGERS.md

```diff
@@ -221,12 +221,6 @@ the file "./VERSION". This binary is usually installed somewhere like
 
 ### Dynamic Daemon / Client-only Binary
```

Comment:
Just remove this section please, and add the link to https://github.com/docker/cli

Reply:
Do you mean the section regarding Dynamic Daemon?

Reply:
client stuff

Reply:
Does this solve it?